PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt

```diff
@@ -239,6 +239,11 @@ class PayPalCheckoutRequest @JvmOverloads constructor(
         }
 
         parameters.put(EXPERIENCE_PROFILE_KEY, experienceProfile)
+
+        amountBreakdown?.let {
+            parameters.put(AMOUNT_BREAKDOWN_KEY, it.toJson())
+        }
```

Comment:
We should maybe consider moving all of these to `let`, but nitpick for now let's be consistent with the null check logic we use for all other params